### PR TITLE
Fix typo in reference to microsoft_graph_resource_id endpoint

### PR DESCRIPTION
**Pull request by psignoret**
Fixes typo in reference to Microsoft Graph endpoint, to align with typo fixes in https://github.com/Azure/azure-cli/pull/1004

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by MyronFanQiu** [comment]
@psignoret  Hello. Could you please update History.rst?

**Comment by qwordy** [comment]
How can we preview the final result of automatic merge? Sometimes the merge result is not what we expect.

**Comment by yonzhan** [comment]
@jiasli  please help deal with this PR.

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -53,6 +53,10 @@ Release History
 
 * az monitor log-analytics workspace: Support CRUD for Azure log analytics workspace.
 
+**Profile**
```
Comment (MyronFanQiu) [inline src/azure-cli/HISTORY.rst]: Move it to the latest version.

**Comment by MyronFanQiu** [comment]
@jiasli Could you please help to update the History again and merge it? I'm ok with this PR.

**Pull request merged**